src/main/resources/templates/fragments/navigation.html

```diff
@@ -0,0 +1,9 @@
+<div th:fragment="nav">
+    <nav>
+        <ul>
+            <li><a href="/">Home</a></li>
+            <li><a href="/blog">Blog</a></li>
+            <li><a href="/login">Login</a></li>
```

Comment:
It's best practice to avoid using file extensions in the path of the URL when using an API controller, so the new navigation component only refers to the html page name. For example using `/blog` instead of `/blog.html`. See [Best Practices for Naming REST API Endpoints](https://blog.dreamfactory.com/best-practices-for-naming-rest-api-endpoints/).